apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -69,44 +69,44 @@ static SessionPool()
         }
 
         // ReSharper disable once UnusedMember.Global
-        // returns true if the pool has evenly split its contents.
+        // returns the maximum difference in size between the smallest and largest pool.
         // For test purposes only.
         // poolcontents will be filled with the contents of the pool and may not be null.
-        internal static bool GetPoolInfo(StringBuilder poolContents)
+        internal static int GetPoolInfo(StringBuilder poolContents)
         {
             GaxPreconditions.CheckNotNull(poolContents, nameof(poolContents));
-            bool result = true;
+            var maxSize = 0;
+            var minSize = int.MaxValue;
             poolContents.AppendLine("SessionPool.Contents (by priority):");
-            int i = 0;
+            var i = 0;
             foreach (var priorityListEntry in s_priorityList.GetSnapshot())
             {
                 poolContents.AppendLine($"SessionPool({i}) Key:${priorityListEntry.Key}"
                     + $" HashCode_of_Pool:{priorityListEntry.GetHashCode()}");
                 i++;
             }
             poolContents.AppendLine("SessionPool.Contents (by client):");
-            int byClientIndex = 0;
-            int firstSize = -1;
+            var byClientIndex = 0;
             foreach (var byClientEntry in s_poolByClientAndDatabase)
             {
                 poolContents.AppendLine($"SessionPool({byClientIndex}) Key:${byClientEntry.Key}"
                     + $" HashCode_of_Pool:{byClientEntry.Value.GetHashCode()}");
                 var size = byClientEntry.Value.DumpSessionPoolContents(poolContents);
                 if (size > 0)
                 {
-                    if (firstSize == -1)
+                    if (minSize > size)
```

Comment:
Could use

```csharp
minSize = Math.Min(size, minSize);
maxSize = Math.Max(size, maxSize);
```

That's more obviously correct IMO - I thought for a while that this was the wrong way round. (But if you disagree, I'm certainly not going to push it.)